Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -684,7 +682,7 @@ class BrowserViewController: UIViewController {
     }
 
     func openURLInNewTab(url: NSURL) {
-        let tab = tabManager.addTab(request: NSURLRequest(URL: url))
+        let tab = tabManager.addTab(NSURLRequest(URL: url))
```

Comment:
Nit: We should rename this to be more swifty like newTabWithRequest or something. Obviously not important for merging this.